packages/ui/src/custom/action-popover/action-popover.svelte

```
@@ -0,0 +1,118 @@
+  async function handleAction() {
+    if (!onAction) return;
+
+    const result = onAction();
+
+    if (result instanceof Promise) {
+      isPending = true;
+
+      try {
+        await result;
+      } finally {
+        return;
```

Biome (2.5.8)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)